web_widget_char_switchcase/static/src/js/web_widget_char_switchcase.js

```diff
@@ -6,6 +6,9 @@ odoo.define('web_widget_char_switchcase', function (require) {
     var form_widgets = require('web.form_widgets');
 
     form_widgets.FieldChar.include({
+        events: _.extend({}, form_widgets.FieldChar.prototype.events, {
+            'keyup': 'key_up_handler',
```

Comment:
good convention: `_on$EVENT_NAME` (w/ or w/out `_`)

Reply:
Done in https://github.com/OCA/web/pull/1079/commits/63e2623b69d22a27b224fe80586e2fef4a9ac99e